website/docs/basics/blocks-component-in-block.md

```diff
@@ -5,3 +5,75 @@ sidebar_label: Component In Block
 ---
 
 [![docs-source](https://img.shields.io/badge/source-eigthshift--frontend--libs-yellow?style=for-the-badge&logo=javascript&labelColor=2a2a2a)](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/blocks/)
+
+Blocks and components go perfectly together because they are made this way. We simplified this as much as possible but there always a few things you need to do manually.
```

Comment:
```suggestion
Blocks and components work perfectly together because they are made that way. We simplified this as much as possible but there always a few things you need to do manually.
```